cvmfs/server/cvmfs_server_util.sh

```diff
@@ -1059,10 +1059,15 @@ _update_geodb() {
         echo "$CVMFS_GEO_DB_FILE doesn't exist or is not readable" >&2
         return 1
       fi
-      rm -f $dbfile
-      echo "Linking GeoIP Database"
-      _to_syslog_for_geoip "linking db from $CVMFS_GEO_DB_FILE"
-      ln -s $CVMFS_GEO_DB_FILE $dbfile
+      if [ $(echo $CVMFS_GEO_DB_FILE | cut --bytes=1-7) = "http://" -o $(echo $CVMFS_GEO_DB_FILE | cut --bytes=1-8) = "https://" ]; then
+        rm -f $dbfile
+        echo "Downloading GeoIP Database"
+        curl $CVMFS_GEO_DB_FILE -o $dbfile
```

Comment:
I'm not sure I Iike this re-use of CVMFS_GEO_DB_FILE.  The CVMFS_GEO_DB_FILE is documented in readthedocs as being a file, not a URL, and the name implies that it is a FILE.  How about just downloading the file in the wrapping test code instead of in cvmfs_server?

I asked @cvuosalo to make an optional way to download a .mmdb file from db-ip.com instead of Maxmind and that's currently just a URL to download from so I think we will get a parameter to set a download URL, but I don't think this is a good way to do it.

Reply:
Fair enough, this is a workaround can be done in the test setup logic, I updated the PR.

Reply:
Having an alternative provider would be very welcome. For our tests it makes sense in any case to have the option to use a locally cached db. I'll configure our weekly builds to fetch from Maxmind so we still have test coverage.